jenkins/catapult/requirements.txt

```diff
@@ -1,2 +1,2 @@
 PyYAML==6.0.1
-node-semver==0.6.1 # use the same version as conan to avoid conflicts
```

Comment:
is this the right requirements or should client/catapult/scripts/requirements.txt also be updated ?

Reply:
yea, it's needed here for the dependency updater scripts.
but probably should also be in ``client/catapult/scripts/requirements.txt``